db/migrate/20151120153254_delete_bootable_interface.rb

```diff
@@ -0,0 +1,11 @@
+class DeleteBootableInterface < ActiveRecord::Migration
+  def up
+    Nic::Base.where(:type => 'Nic::Bootable').each do |nic|
```

Comment:
Doesn't this still have the same issue with all logic that is present in Nic::Managed and its parents?